packages/shopify-cli-extensions/src/configs.ts

```diff
@@ -45,9 +45,11 @@ const RESERVE_PATHS = flattenPaths({
   },
 });
 
-export function getConfigs() {
+const STDIN = 0;
+
+export function getConfigs(source: string | number = STDIN) {
```

Comment:
Should we just to remove support for file paths completely? It would make our lives easier. Otherwise my suggestion would be to document that you need to provide a path to YAML file.